pyproject.toml

```diff
@@ -1,6 +1,76 @@
 [build-system]
 requires = [
-    "setuptools>=42",
-    "wheel"
+    "setuptools>=67",
+    "wheel",
 ]
 build-backend = "setuptools.build_meta"
+
+# https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+[project]
+name = "urbanopt-ditto-reader"
+dynamic = [
+    "readme",
+    ]
+version = "0.5.2"
+authors = [
+  { name="Tarek Elgindy", email="tarek.elgindy@nrel.gov" },
+  { name="Nathan Moore", email="nathan.moore@nrel.gov" },
+  { name="Katherine Fleming", email="katherine.fleming@nrel.gov" },
+]
+description = "Enhancement of URBANopt GeoJSON that can be consumed by DiTTo reader"
+requires-python = "~=3.7"
+classifiers = [
+        "Development Status :: 4 - Beta",
+        "Environment :: Console",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "Topic :: Scientific/Engineering",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 3.10"
+    ]
+dependencies = [
+    "ditto.py[opendss]~=0.2",
+    "opendssdirect.py ~= 0.7"
```

Comment:
Just checking - did we want to require version 0.8 in this installation rather than 0.7? It might help streamline our tests with the latest opendssdirect version.

Reply:
I will remove the opendssdirect.py change from this PR and make the change only in #48.